daffodil-lib/src/main/resources/org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd

```diff
@@ -19,101 +19,17 @@
 <xs:schema xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
   xmlns:xs="http://www.w3.org/2001/XMLSchema">
 
+  <xs:include schemaLocation="DFDLGeneralFormatBase.dfdl.xsd" />
+
   <!-- Intended for inclusion into other schemas. Has no targetNamespace on purpose -->
   <xs:annotation>
     <xs:appinfo source="http://www.ogf.org/dfdl/">
 
-      <!--
-        This set of properties should never change. It can be added to 
-        if properties are missing, but should not be changed.
-        
-        Changes should be introduced by way of defining new formats in
-        terms of this one, which override definitions. See below in 
-        the file for examples. 
-       -->
-      <dfdl:defineFormat name="GeneralFormatOriginal">
```

Comment:
I see GeneralFormatOriginal has been moved to its own file called DFDLGeneralFormatBase.dfdl.xsd and that file included above here.